tests/Unit/Crypt/BlowfishTest.php

```diff
@@ -77,7 +77,15 @@ public function testVectors($engine, $key, $plaintext, $expected)
         $bf = new Blowfish('cbc');
         $bf->setKey($key);
         $bf->setIV(str_repeat("\0", $bf->getBlockLength() >> 3));
-        if (!$bf->isValidEngine($engine)) {
+
+        $engines = array(
+            'PHP'=>\phpseclib\Crypt\Common\SymmetricKey::ENGINE_INTERNAL,
+            'Eval'=>\phpseclib\Crypt\Common\SymmetricKey::ENGINE_EVAL,
+            'mcrypt'=>\phpseclib\Crypt\Common\SymmetricKey::ENGINE_MCRYPT,
+            'OpenSSL'=>\phpseclib\Crypt\Common\SymmetricKey::ENGINE_OPENSSL,
+        );
+
+        if (!$bf->isValidEngine($engines[$engine])) {
```

Comment:
I think this is a bad change. Right now there are 206 skipped unit tests:

https://travis-ci.org/phpseclib/phpseclib/jobs/281628707

After this commit there will be 308 tests:

https://travis-ci.org/phpseclib/phpseclib/jobs/274496080

In https://github.com/phpseclib/phpseclib/commit/9d8f014748e4787c4f54a1bb8070cc4c8bb44497#comments I updated phpseclib to take strings instead of ints. This change was inspired by another PR you made - #1164. With this change you're changing what's passed to `isValidEngine` back from strings to ints.

Reply:
Never mind - I see that you undid this lol. Sorry!